.github/workflows/build.yaml

```diff
@@ -43,6 +46,7 @@ jobs:
             # disabled if major zero
             type=semver,pattern={{major}},enable=${{ !startsWith(github.ref, 'refs/tags/v0.') }}
             type=ref,event=branch
+            type=ref,event=pr
```

Comment:
this also adds the tag to release versions?

Reply:
No, that is configured above. See https://github.com/docker/metadata-action for some explanation.

Reply:
Current tags: https://github.com/statnett/image-scanner-operator/pkgs/container/image-scanner-operator